include/functions.h

```diff
@@ -1112,8 +1119,8 @@ u32 func_800C9D8C(CollisionContext* param_1, CollisionPoly* param_2, s32 param_3
 u32 func_800C9E18(CollisionContext* bgCtxt, CollisionPoly* polygon, s32 index);
 u32 func_800C9E40(CollisionContext* bgCtxt, CollisionPoly* polygon, s32 index);
 u32 func_800C9E88(CollisionContext* bgCtxt, CollisionPoly* polygon, s32 index);
-// void func_800C9EBC(UNK_TYPE1 param_1, UNK_TYPE1 param_2, UNK_TYPE1 param_3, UNK_TYPE1 param_4, UNK_TYPE4 param_5, UNK_TYPE4 param_6, UNK_TYPE4 param_7);
-// void func_800CA1AC(UNK_TYPE1 param_1, UNK_TYPE1 param_2, UNK_TYPE1 param_3, UNK_TYPE1 param_4, UNK_TYPE4 param_5, UNK_TYPE4 param_6);
+// void func_800C9EBC(GlobalContext* globalCtx, CollisionContext* bgCtxt, UNK_TYPE1 param_3, UNK_TYPE1 param_4, UNK_TYPE4 param_5, UNK_TYPE4 param_6, UNK_TYPE4 param_7);
```

Comment:
can all cases of `bgCtxt` be renamed to `colCtx`?

Reply:
There's a few contexts that are slightly inconsistent in naming but sometimes it falls outside of `functions.h` as well. It's a todo item that may or may not be part of this pr depending on how Anghelo wants to do. Some of the names I've noticed involves:
- `ActorContext*`: `actCtxt`/`aCtxt` -> `actorCtx`
- `CollisionContext`: bgCtxt` -> `colCtx`
- `TitleCardContext*`: `titleCtxt` -> `titleCardCtx`
- `TargetContext*`: `targetCtxt` -> `targetCtx`
- `GameState*`: `ctxt` -> `gamestate`
- `KankyoContext*`: `kanCtxt` -> `kanCtx`
- `LightContext*`: `lCtxt` -> `lightCtx`
- `RoomContext*`: `roomCtxt` -> `roomCtx`
- `MtxF*`: `ctxt` -> `mtx`
- `TitleContext*` -> `ctxt` -> `titleCtx`

Reply:
kanctx is supposed to be envCtx (EnvironmentContext), I thought I had already renamed that too months ago 🤔

Reply:
Done. Thanks!